packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -155,6 +155,17 @@ export class RelationshipsController extends TransportController {
             throw this.newCacheEmptyError(RelationshipTemplate, template.id.toString());
         }
 
+        const queryForExistingRelationship = {
+            "peer.address": template.cache.createdBy.toString(),
+            status: { $in: [RelationshipStatus.Pending, RelationshipStatus.Active, RelationshipStatus.Terminated, RelationshipStatus.DeletionProposed] }
```

Comment:
Do I see it correctly, that this line won't be needed, if the idea of @jkoenig134 is implemented to directly delete rejected and revoked relationships?

Reply:
yeeeah :)